pkg/jobrunaggregator/jobrunhistoricaldataanalyzer/analyzer.go

```diff
@@ -224,12 +224,11 @@ func (o *JobRunHistoricalDataAnalyzerOptions) compareAndUpdate(newData, currentD
 	}
 
 	return compareResults{
-		increaseCount:   increaseCountP99,
-		decreaseCount:   decreaseCountP99,
-		addedJobs:       added,
-		jobs:            results,
-		missingJobs:     missingJobs,
-		newReleaseEvent: newReleaseEvent,
+		increaseCount: increaseCountP99,
+		decreaseCount: decreaseCountP99,
```

Comment:
I still haven't found the bug we talked about on call yet, but it's because increaseCount and decreaseCount are both coming out 0. Everything is classified as an "added" job, and nothing is getting compared as a result, thus nothing is increased/decreased. if increasedCount is 0, we have no table to display.